src/components/Modals/OnboardingReview.vue

```diff
@@ -0,0 +1,514 @@
+      const sections = ['type', 'subject', 'score'];
+      if ('exam' in this.user && this.user.exam.length > 0) {
+        for (let x = 0; x < this.user.exam.length; x += 1) {
+          const exam = {};
+          for (const sec of sections) {
+            exam[sec] = {
+              shown: false,
+              stopClose: false,
+              boxBorder: '',
+              arrowColor: '',
+              placeholderColor: '#757575',
+              placeholder: this.user.exam[x][sec],
+              acronym: '',
+            };
+          }
+          if (typeof this.user.exam[x].subject !== 'undefined') {
+            exams.push(exam);
+            credits += this.user.exam[x].credits;
+            exam.equivCourse = this.user.exam[x].equivCourse;
+          }
+        }
+      }
+      const exam = {};
+      for (const sect of sections) {
+        exam[sect] = {
+          shown: false,
+          stopClose: false,
+          boxBorder: '',
+          arrowColor: '',
+          placeholderColor: '',
+          placeholder: placeholderText,
+          acronym: '',
+        };
+      }
+      exams.push(exam);
+      this.displayOptions.exam = exams;
+      const swim = typeof this.user.tookSwim !== 'undefined' ? this.user.tookSwim : 'no';
+      this.tookSwimTest = swim;
+      const transferClass = [];
+      this.user.transferCourse.forEach(course => {
+        transferClass.push(course);
+      });
+      transferClass.push({ class: placeholderText, credits: 0 });
+      this.displayOptions.class = transferClass;
+    },
+    getCredits() {
+      let count = 0;
+      this.displayOptions.exam.forEach(exam => {
+        if (this.transferJSON !== null) {
+          const name = exam.subject.placeholder;
+          if (name in this.transferJSON) {
+            count += this.transferJSON[name].credits;
+          }
+        }
+      });
+      this.displayOptions.class.forEach(clas => {
+        count += clas.credits;
+      });
+      this.totalCredits = count;
+    },
+    getExamCredit(exam) {
+      const name = exam.subject.placeholder;
+      if (this.transferJSON !== null) {
+        if (name in this.transferJSON) {
+          return this.transferJSON[name].credits;
+        }
+      }
+      return 0;
+    },
+    getTransferMap() {
+      const TransferJSON = {};
+      examData.AP.forEach(sub => {
+        TransferJSON[sub.name] = {
+          credits: sub.fulfillment.credits,
+          type: 'AP',
+        };
+      });
+      examData.IB.forEach(sub => {
+        TransferJSON[sub.name] = {
+          credits: sub.fulfillment.credits,
+          type: 'IB',
+        };
+      });
+      this.transferJSON = TransferJSON;
+      if (typeof this.displayOptions !== 'undefined') {
+        this.$emit(
+          'updateTransfer',
+          this.displayOptions.exam,
+          this.displayOptions.class,
+          this.tookSwimTest
+        );
+      }
+    },
+    // Set the colleges map to with acronym keys and full name values
```

Comment:
Can you fix the comment here and the next function? You changed it in another vue file but not this one haha

Reply:
Oh did I delete it in another file? I have no idea what that comment means

Reply:
Nevermind I got it and it's fixed